bitbucket/mixins.py

```diff
@@ -0,0 +1,50 @@
+from django.shortcuts import get_object_or_404
+from django.utils.functional import cached_property
+
+from guardian.mixins import PermissionRequiredMixin
+from rest_framework.views import APIView
+
+from . import serializers
+from .constants import API_TO_MODEL_MAP
+from .exceptions import bitbucket_exception_to_api
+from .models import KatkaProject
+
+
+class BitbucketPermissionRequiredMixin(PermissionRequiredMixin, APIView):
+    permission_required = 'view_katkaproject'
+
+    queryset = KatkaProject
+    lookup_fields = ('katka_project_id',)
+    lookup_fields_serializer = serializers.KatkaProjectSerializer
```

Comment:
why renaming to `KatkaProjectSerializer`?

Reply:
When importing serializer.KatkaProject and models.KatkaProject they have the same, thought about changing the name of it to include Serializer, but I could also make a more specific name for the import.